src/registry/selectboxes/transform-data.tsx

```diff
@@ -0,0 +1,26 @@
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {Checkbox} from '@/components/formio';
+
+const TransformData = () => {
+  const intl = useIntl();
+  const tooltip = intl.formatMessage({
+    description: "Tooltip for 'Transform data' builder field",
+    defaultMessage: `Transform the submitted data to a list with the selected choices, instead of an
+      object that maps choices to true or false (dependent on whether they are selected).`,
+  });
+  return (
+    <Checkbox
+      name="openForms.transformData"
+      label={
+        <FormattedMessage
+          description="Label for 'Transform data' builder field"
```

Comment:
```suggestion
          description="Label for selectboxes 'Transform data' builder field"
```
probably a good idea to "namespace" this :)